traits/tests/test_enum.py

```diff
@@ -296,3 +302,25 @@ def test_base_enum(self):
         with self.assertRaises(TraitError):
             obj.slow_enum = "perhaps"
         self.assertEqual(obj.slow_enum, "no")
+
+
+class TestGui(GuiTestAssistant, unittest.TestCase):
+
+    def test_create_editor(self):
+        obj = EnumCollectionGUIExample()
+        traits = obj.class_trait_names()
+        traits.remove("trait_added")
+        traits.remove("trait_modified")
+
+        ui = obj.edit_traits()
+        for t in traits:
+
+            with self.subTest(t=t):
+                editor = getattr(ui.info, t)
+
+                # Try setting all valid values for the Enum trait
+                for value in obj.trait(t).trait_type.values:
+                    with self.assertTraitChangesInEventLoop(
+                            obj, t, lambda instance: getattr(
+                                instance, t) == value, 0, 3):
+                        self.gui.set_trait_later(editor, 'value', value)
```

Comment:
@corranwebster I don't think this is the right way to emulate the selection of an item from a drop-down menu. Do you know off the top of your head, if there some way I could do that ?

Reply:
I even tried setting the `names` in the `editor`, but those are strings and won't work.